IntegrationAdaptersTests/RegisterHospitalTests.cs

```diff
@@ -21,20 +21,20 @@ public void Is_pharmacy_already_existing_on_hospital()
             ApiService service = new ApiService();
 
             Api api = new Api("k", "k", "k");
-            bool result = service.RegisterHospitalOnPharmacy(api);
+            bool result = service.IsPharmacyExistsOnHospital(api);
```

Comment:
💯 Are these the tests that failed if you run them again? 😄 This should work

Reply:
Yes, but now it is pass every time.
Thanks for suggestion 😄